metric/system/process/process_windows.go

```diff
@@ -436,3 +453,23 @@ func getIdleProcessMemory(state ProcState) (ProcState, error) {
 	}
 	return state, nil
 }
+
+func fillIdleProcess(state ProcState) (ProcState, error) {
+	state, err := getIdleProcessMemory(state)
+	if err != nil {
+		return state, err
+	}
+	total0, idle0, err := getIdleProcessTime()
+	if err != nil {
+		return state, err
+	}
+	time.Sleep(500 * time.Millisecond)
```

Comment:
Do we have to Sleep?  I might be missing something but I think this means we will wait 1/2 sec every time we try to get process stats on windows, because PID 0 will always be in the list.

Reply:
Well, I'm doing this to calculate the percentage in which CPU was idle. 
I think we can reduce the timeout. 1/2 second is bit too much.

Reply:
@leehinman i've used `50s` as a timeout value.